tools/partitions/app3M_fat9M_16MB.csv

```diff
@@ -4,4 +4,5 @@ otadata,  data, ota,     0xe000,  0x2000,
 app0,     app,  ota_0,   0x10000, 0x300000,
 app1,     app,  ota_1,   0x310000,0x300000,
 ffat,     data, fat,     0x610000,0x9F0000,
-# to create/use ffat, see https://github.com/marcmerlin/esp32_fatfsimage
+coredump, data, coredump, , 64K,
+# to create/use ffat, see https://github.com/marcmerlin/esp32_fatfsimage
```

Comment:
if you do not lower the size of the last partition, coredump will end up outside of the flash chip

Reply:
Would leaving it blank cause it to automatically take up the rest of the space?

Reply:
afraid not. You do need to subtract 64K from the size